src/System.ComponentModel.TypeConverter/tests/Int16ConverterTests.cs

```diff
@@ -2,44 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Globalization;
-using Xunit;
 
 namespace System.ComponentModel.Tests
 {
-    public class Int16ConverterTests : ConverterTestBase
+    public class Int16ConverterTests : TypeConverterTestBase
     {
-        private static TypeConverter s_converter = new Int16Converter();
+        public override TypeConverter Converter => new Int16Converter();
 
-        [Fact]
-        public static void ConvertFrom_WithContext()
+        public override IEnumerable<ConvertTest> ConvertToTestData()
         {
-            ConvertFrom_WithContext(new object[3, 3]
-                {
-                    { "-1  ", (short)(-1), null },
-                    { "#2", (short)2, null },
```

Comment:
Are you moving the test for `{ "#2", (short)2, null },` to somewhere else?

Reply:
Fixed